### PR TITLE
Milestone 5: Contact form mobile version

### PR DESCRIPTION
**Pull request by Cavein254**
This milestone accomplishes the following:
- Create a contact form
- Add HTML validation for name, email, and message fields

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -342,5 +342,33 @@ <h5>Skills</h5>
...
+    <div class="form-container">
+      <div class="form-header">
+        <h4>Contact me </h4>
+      </div>
+      <div class="form-desc">
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
+      </div>
+      <div class="form-input" >
+      <div class="form-val">
+        <input type="text" placeholder="Name" minlength="5" maxlength="30" required/>
```
Comment (franclobo) [inline index.html]: - [x] Although I filled all the required inputs, it returned an error 👇🏽 

image

Please 🙏🏽 fix it for a better user experience.

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Cavein254, 

# Required Changes ♻️

so-close-jim-halpert

Good job so far!
You are doing it well 👏🏽 there are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Correct attributes on HTML inputs 💯 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Cavein254) [reply index.html]: Done :heavy_check_mark:

**Review by divinecharlotte — APPROVED** [review]
# Hello @Cavein254   😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 
### To Highlight 💯

- No linters errors ✔️
- Descriptive PR ✔️






## Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**